package.json

```diff
@@ -17,12 +17,11 @@
     "automated messaging"
   ],
   "author": {
-    "name": "Artur Ataide",
-    "url": "https://arturataide.io"
+    "name": "Lukas Schwab"
```

Comment:
Hey Lukas,

There is the possibility to add multiple authors without removing the credits of someone who spent some time on this :)

Reply:
Hey Artur,

Sorry for taking so long to answer. I want to apologize, it was never my intention to remove you as an author on your repository. After studying your code, I realized that it would be much simpler to rewrite the out-of-office bot from scratch. You version uses the @slack/bolt library which is a big library intended for building big slack apps permanently rûnning on a server. It has a lot of functionality that the out-of-office bot doesn't need, especially because it's deployed as an AWS Lambda function. Also, the only thing you are using from the bolt library is the chat.postMessage API method. This is available in its own npm module @slack/web-api.

Reply:
Also, the reason why the bot didn't work is that you handled every message event by the Slack Events API as a user message. In reality, every action that has something to do with messages such as posting a message, editing a message, reacting to a message, deleting to a message, etc. is dispatched as an event with { "type": "message" }. You have to check the subtype of these events to know which of these events are actual messages that were posted and which of them are irelevant for the bot and should be ignored. You can read more about that here: [https://api.slack.com/events/message](https://api.slack.com/events/message).

Reply:
However, it was my mistake because I didn't want to send the pull request to your repository. I mixed them up because I named the repo on TX side similarly, sorry for that. Feel free to close this PR.

Also I added you as an author in the package.json file of the new repo. Sorry for the confusion, I understand that this looked like I took your work and credited it as my own but that was never my intention.